### ISSUE TITLE
[Tracking] AccessibiltyExtensions.cs is misspelled

### ISSUE THREAD

**Issue by paul1956** [dotnet/winforms#12155]
It should be AccessibilityExtensions and file name changed to AccessibilityExtensions.cs

**Comment by paul1956** [comment]
Fixed by #12188
